massa-execution/src/interface_impl.rs

```diff
@@ -155,4 +158,22 @@ impl Interface for InterfaceImpl {
             .map(|addr| addr.to_bs58_check())
             .collect())
     }
+
+    fn generate_event(&self, data: String) -> Result<()> {
+        let slot = context_guard!(self).slot;
+        let bytes_address = *context_guard!(self).call_stack.back().unwrap(); // TODO what's an empty stack ?
+        let caller_address = *context_guard!(self).call_stack.front().unwrap();
+        let block = context_guard!(self).opt_block_id;
+        let call_stack = context_guard!(self).call_stack.clone();
+        let context = EventExecutionContext {
+            slot,
+            bytes_address,
```

Comment:
Let's remove the entries "bytes_address" and "caller_address" from EventExecutionContext. It's redundant with call_stack